# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/ulimit.h-stub.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We just place a stub ulimit.h in the dietlibc include directory to get stuff
like pdksh to compile.

  - Rene Rebe <rene@exactcode.de>

--- dietlibc-0.27/include/ulimit.h.orig	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.27/include/ulimit.h	2004-10-02 17:45:44.051302108 +0200
@@ -0,0 +1,6 @@
+#ifndef _ULIMIT_H_
+#define _ULIMIT_H_
+
+/* just a stub -ReneR */
+
+#endif /* _ULIMIT_H_ */
